src/components/document/sorted-section.tsx

```diff
@@ -2,21 +2,22 @@ import React, { useState } from "react";
 import { observer } from "mobx-react";
 import classNames from "classnames";
 
-import { DocumentModelType, getDocumentContext } from "../../models/document/document";
-import { DecoratedDocumentThumbnailItem } from "../thumbnail/decorated-document-thumbnail-item";
+import { DocumentModelType } from "../../models/document/document";
 import { useStores } from "../../hooks/use-stores";
-import { logDocumentViewEvent } from "../../models/document/log-document-event";
-import { ENavTab } from "../../models/view/nav-tabs";
 import { DocFilterType, SecondarySortType } from "../../models/stores/ui-types";
-import { SimpleDocumentItem } from "../thumbnail/simple-document-item";
 import { IDocumentMetadata } from "../../../functions/src/shared";
-import { DocumentContextReact } from "./document-context";
-import { DocumentCollection } from "../../utilities/sort-document-utils";
 import { DocumentGroup } from "../../models/stores/document-group";
+import { DocumentGroupComponent } from "./document-group";
+import { logDocumentViewEvent } from "../../models/document/log-document-event";
+import { DecoratedDocumentThumbnailItem } from "../thumbnail/decorated-document-thumbnail-item";
+import { ENavTab } from "../../models/view/nav-tabs";
 
 import ArrowIcon from "../../assets/icons/arrow/arrow.svg";
 
+// TODO: Figure out how to totally move sorted-section-specific styles out of sort-work-view.scss without
+// breaking the layout. The focus document header and close button are being affected.
```

Comment:
I've removed this todo comment. While trying to explain why I left it, I finally realized what the problem was. I had been confused about why we had to import sort-work-view.scss into the `SortedSection` component to prevent the layout from breaking, since the component is a child component of `SortWorkView`. Turns out sort-work-view.scss wasn't being imported into `SortWorkView`. I should've caught that earlier.